guardian/media-atom-maker
# Bump webpack to v5 (and bump other packages) to remove vulns

## What does this change?

This PR bumps Webpack to v5, in order to bump `ts-loader` - which was responsible for the last high severity vulnerabilities.

![image](https://user-images.githubusercontent.com/34686302/219620604-ad6084c7-2cda-4edd-b2d9-a7d0ff3de107.png)

There will now be only one vulnerability left in the application, a medium severity vulnerability from fork-ts-checker-webpack-plugin, which is yet to be solved in the source package.

n.b -  the `client-dev.sh` script and `yarn client-dev` (the hot-loading scripts) aren't working here, but they aren't working on main either (it looks like no built artifacts are created, and the dev-server consequently fails to find them). I have made sure not to introduce any new problems to the build process in those scripts though.

## How to test

1. Run this project according to the instructions in the readme. Does it build and run as expected?
2. Separately deploy to CODE (to test `webpack.prod.conf.js` 

## How can we measure success?

- The project builds, deploys and runs successfully. 
- The high priority vulnerabilities are fixed.
